packages/app-bridge/src/utilities/guideline.ts

```diff
@@ -0,0 +1,13 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Document, DocumentGroup } from '../types';
+
+export const mergeSort = (
```

Comment:
Please make utility generic and name it `mergeSort` and add tests

Reply:
this will be deleted. An old implementation that is not needed.